### DISCUSSION TITLE
How to change app icon?

### DISCUSSION THREAD

**Discussion by fairking** [wailsapp/wails#3214] [General]
I compile on Windows with the following command:
```bash
wails build
```

Is there any way to change the app/window icon?

![image](https://github.com/wailsapp/wails/assets/13495631/49275e8b-6041-407b-b718-9eb172ae52b0)

Cannot find anything in [docs](https://wails.io/docs/introduction).

**Comment by fairking** [comment]
On Linux it look like this:
![image](https://github.com/wailsapp/wails/assets/13495631/dbf49add-7a18-4f14-9e3b-eb3eec7d57ca)

**Comment by ansxuman** [comment]
> I compile on Windows with the following command:
> 
> ```shell
> wails build
> ```
> 
> Is there any way to change the app/window icon?
> 
> ![image](https://private-user-images.githubusercontent.com/13495631/298286335-49275e8b-6041-407b-b718-9eb172ae52b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU3NzkyMzYsIm5iZiI6MTcwNTc3ODkzNiwicGF0aCI6Ii8xMzQ5NTYzMS8yOTgyODYzMzUtNDkyNzVlOGItNjA0MS00MDdiLWI3MTgtOWViMTcyYWU1MmIwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIwVDE5Mjg1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwOTMxOWNkYTcxOTBjOGYxZTNjZDRkMGNjODQ1ZWU2MTQyNWI1MzcyOTFkZmRiNzc0ZjYxMzkwMzc2MjQ3OTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DGivbYSSKXw-qjb7iIXaSshwXdXLea3jq5AtRb8nB7U)
> 
> Cannot find anything in [docs](https://wails.io/docs/introduction).

On build dir, replace the `AppIcon.png` with yours to get your desired app icon

**Reply by fairking** [reply]
Thank you. I managed to change the icon on Windows. But so far on Linux the icon remains the same.

**Reply by leaanthony** [reply]
Let us know when Linux has a way of doing that 😅 I believe the only way currently is to update your `.desktop` file if you have one
